app/azure/Dashboard/index.ts

```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+
+import { DashboardViewAngular } from './DashboardView';
+
+export const dashboardModule = angular
+  .module('portainer.azure.dashboard', [])
```

Comment:
I think this view doesn't need a whole new module for it. just import it in the main azure module

module is required if we have more than one (or a few) components, if it's only one, just import to the main module

Reply:
done

Reply:
actually i couldn't figure this out, modules in angular is something i havent done much of before 😬

Reply:
just move line 7 to `app/azure/__module.js`